lib/update-v8/gypfiles.js

```diff
@@ -5,7 +5,11 @@ const path = require('path');
 const fs = require('fs-extra');
 
 function nodeOwnsGypfiles(ctx) {
-  if (ctx.currentVersion[0] === 6 && ctx.currentVersion[1] < 6) {
+  if (
```

Comment:
```suggestion
  const hasGypFiles = fs.exists(path.join(ctx.nodeDir, 'deps/v8/gypfiles'));
  if (!hasGypFiles) {
    return false;
  }
  const [v8_major, v8_minor, ...] = ctx.currentVersion;
  if (v8_major >= 7 || (v8_minor === 6 && v8_minor > 5)) {
    return true;
  }
  return false;
```

Reply:
I went for a more drastic change and removed this code entirely. We won't do major upgrades anymore on the current release lines so in the future gypfiles will always be in `tools`

Reply:
> removed this code entirely.

💚